reloc/python3/build_reloc.sh

```diff
@@ -32,5 +32,5 @@ PYVER=$(python3 -V | cut -d' ' -f2)
 echo "$PYVER" > build/python3/SCYLLA-VERSION-FILE
 ln -fv build/SCYLLA-RELEASE-FILE build/python3/SCYLLA-RELEASE-FILE
 
-PACKAGES="python3-pyyaml python3-urwid python3-pyparsing python3-requests python3-pyudev python3-setuptools python3-psutil"
+PACKAGES="python3-pyyaml python3-urwid python3-pyparsing python3-requests python3-pyudev python3-setuptools python3-psutil python3-distro"
```

Comment:
Need to add to install-dependency.sh too

Reply:
I've check the image, and it's already installed, but I'll add it to make it explicit

Reply:
added it in be8dd05